README.md

```diff
@@ -25,7 +25,10 @@ By making Event Hubs and Spark easier to use together, we hope this connector ma
 |Spark Version|Package Name|Package Version|
 |-------------|------------|----------------|
 |Spark 2.3|azure-eventhubs-spark_2.11|[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.microsoft.azure/azure-eventhubs-spark_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.microsoft.azure/azure-eventhubs-spark_2.11)|
-|Spark 2.2|azure-eventhubs-spark_2.11|[![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.azure/azure-eventhubs-spark_2.11/2.2.0.svg)](https://maven-badges.herokuapp.com/maven-central/com.microsoft.azure/azure-eventhubs-spark_2.11)|
+|Spark 2.2|azure-eventhubs-spark_2.11|[![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.azure/azure-eventhubs-spark_2.11/2.2.0-PREVIEW.svg)](https://maven-badges.herokuapp.com/maven-central/com.microsoft.azure/azure-eventhubs-spark_2.11)|
+|Spark 2.1|azure-eventhubs-spark_2.11|[![Maven Central](https://img.shields.io/maven-central/v/com.microsoft.azure/azure-eventhubs-spark_2.11/2.2.0-PREVIEW.svg)](https://maven-badges.herokuapp.com/maven-central/com.microsoft.azure/azure-eventhubs-spark_2.11)|
+
+<sup>*Note:* Version 2.2.0 will supports Spark 2.1 as well.</sup>
```

Comment:
1. s/will supports/will support
2. What's the purpose of the note? I don't understand it.

Reply:
Well, 2.3.0 does *not* work with older versions of Spark while 2.2.0 will work with Spark 2.1 and Spark 2.2. The table should be sufficient though, so I'll drop the note for now.